### DISCUSSION TITLE
v5.6.0

### DISCUSSION THREAD

**Discussion by niden** [phalcon/cphalcon#16528] [General]
## What's Changed
* T16371 postgres fix by @niden in https://github.com/phalcon/cphalcon/pull/16509
* Update Router.zep by @zikezhang in https://github.com/phalcon/cphalcon/pull/16328
* #16337 - Adjust Validation:validate() signature by @Jeckerson in https://github.com/phalcon/cphalcon/pull/16338
* T16514 find with source by @niden in https://github.com/phalcon/cphalcon/pull/16516
* fixed skipping application/x-www-form-urlencoded in getPut() by @sinbadxiii in https://github.com/phalcon/cphalcon/pull/16520
* add backer by @Arhell in https://github.com/phalcon/cphalcon/pull/16517
* T16263 deprecation warnings by @niden in https://github.com/phalcon/cphalcon/pull/16525

## New Contributors
* @zikezhang made their first contribution in https://github.com/phalcon/cphalcon/pull/16328

**Full Changelog**: https://github.com/phalcon/cphalcon/compare/v5.6.0...v5.6.1

<hr /><em>This discussion was created from the release <a href='https://github.com/phalcon/cphalcon/releases/tag/v5.6.1'>v5.6.0</a>.</em>

**Comment by lubber-de** [comment]
Where (again) are the windows builds for 8.1, 8.2, 8.3 ?

**Reply by blackjyn** [reply]
Windows builds are there already

**Comment by prachipachpande** [comment]
Builds for 8.2 is not working for me in xampp version 8.2.12
error message I received:
PHP Warning: PHP Startup: Unable to load dynamic library 'php_phalcon.dll' (tried: \xampp\php\ext\php_phalcon.dll (The specified module could not be found), \xampp\php\ext\php_php_phalcon.dll.dll (The specified module could not be found)) in Unknown on line 0
PHP Warning: PHP Startup: psr: Unable to initialize module\nModule compiled with module API=20210902\nPHP compiled with module API=20220829\nThese options need to match\n in Unknown on line 0
